frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -120,6 +120,8 @@ export function convertUniversalFiltersToRecordingsQuery(universalFilters: Recor
             actions.push(f)
         } else if (isLogEntryPropertyFilter(f)) {
             console_log_filters.push(f)
+        } else if (isHogQLPropertyFilter(f)) {
```

Comment:
this "just works"™

Reply:
Does this work for all property types e.g person & event properties? Wondering if the subqueries in the runner affect things here in any way

Reply:
I would be more worried about it not working for folks without PoE enabled (but maybe that's everyone now)